README.md

```diff
@@ -9,6 +9,14 @@ with [Rocket](https://rocket.rs/).
 _ferrocarril_ means _railway_ in Spanish and sounds like _ferrous_ which means
 _containing iron_.
 
+## Usage
+
+```bash
+cargo run --bin foolsgold
```

Comment:
Thank you for adding this. Would you be up for adding the same thing for the `rirb` binary from the `mruby-bin` crate?

Reply:
Yes but I'm not sure what we want me to add here exactly 😅

Reply:
Filed GH-35

Reply:
this got fixed in GH-50 💪